go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/PingCAP-QE/NaglfarCloud
 go 1.16
 
 require (
-	k8s.io/api v0.19.8
-	k8s.io/apimachinery v0.19.9-rc.0
-	k8s.io/client-go v0.19.8
-	k8s.io/component-base v0.19.8
+	k8s.io/api v0.20.2
```

Comment:
v0.19.8 doesn't work?  I recommend to uses this version since most of our k8s cluster use it.

Reply:
Actually, this version is unused as we replace it with v0.19.8

Reply:
But it may be confusing, I will fix it.